### DISCUSSION TITLE
after installing SQLAlchemy 2.0, update statement is not updating the field in DB

### DISCUSSION THREAD

**Discussion by quentinjs** [sqlalchemy/sqlalchemy#9001] [Usage Questions] [answered]
### Describe the bug

The update call below does not actually update the field in the new 2.X release.  As soon as I reverted to thee 1.X the update worked perfectly. 

Windows 10
Using MySQL.  
Python 3.10

I have a table with 3 fields.
id  int
Name varchar
Junk1 varchar

I try and do an update and it finds the record, but doesn't update the field.    keyname is test.c.id  and keyvalue is a valid id value

 

### To Reproduce

```python
stmt = tablename.update().where(keyname == keyvalue).values(Junk1 = 'foo')
    res = conn.execute(stmt)
```

### Error

```
# Copy complete stack trace and error message here, including SQL log output if applicable.
``` 

### Versions

- OS: Windos 10
- Python: 3.10
- SQLAlchemy: 2,X
- Database: MySQL
- DBAPI (eg: psycopg, cx_oracle, mysqlclient):

**Comment by zzzeek** [comment, marked as answer]
hi there -

you are not committing your transaction.  please see https://docs.sqlalchemy.org/en/20/changelog/migration_20.html#library-level-but-not-driver-level-autocommit-removed-from-both-core-and-orm
